### DISCUSSION TITLE
How to adjust `font.getbbox` to use top to bottom as a text height

### DISCUSSION THREAD

**Discussion by AlexanderPodorov** [python-pillow/Pillow#8473] [Q&A] [answered]
Right now by default the text height is calculated from ascender to descender. See screenshot from docs below.
Is it possible to use top to bottom instead?
In red - default behavior
In blue - what is needed
![image](https://github.com/user-attachments/assets/b0b7bf2c-0c9e-4162-b722-b35da1b003e7)

**Comment by radarhere** [comment, marked as answer]
Does this suit your needs?
```python
from PIL import Image, ImageFont

font = ImageFont.truetype("Tests/fonts/FreeMono.ttf", size=48)
left, top, right, bottom = font.getbbox("Quick")
width, height = right - left, bottom - top
print(width, height)
```
It prints a height of 35, which matches the height you expect from
```python
from PIL import Image, ImageDraw, ImageFont
font = ImageFont.truetype("Tests/fonts/FreeMono.ttf", size=48)
im = Image.new("RGB", (150, 50))
draw = ImageDraw.Draw(im)
draw.text((0, 0), "Quick", font=font)
im.save("out.png")
```
![out](https://github.com/user-attachments/assets/7663d76e-dee4-4cee-b833-7c55331c03b7)

**Reply by AlexanderPodorov** [reply]
Thank you, it did not solve my issue completely, but using this approach I was able to get good enough result.
